src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/entity/TestData.kt

```diff
@@ -0,0 +1,14 @@
+package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.entity
+
+import javax.persistence.Entity
+import javax.persistence.Id
+import javax.persistence.Table
+
+@Entity
+@Table(name = "test_data")
```

Comment:
instead of having a test_data table why not have one of the real; tables with minimal columns; e.g. `licence`